### PR TITLE
add facebooks to entertainment

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -68,36 +75,42 @@ websites:
     - name: Plex
       url: https://www.plex.tv
       twitter: plexapp
+      facebook: TVPlex
```
Comment (Jawshy) [inline _data/entertainment.yml]: Their website links to the [plexapp](https://www.facebook.com/plexapp) Facebook page. Where did you find [TVPlex](https://www.facebook.com/TVPlex)?

File: _data/entertainment.yml [hunk 2]
```diff
@@ -30,30 +32,35 @@ websites:
...
       url: http://www.last.fm/
       twitter: lastfmsupport
+      facebook: lastfm
       img: lastfm.png
       tfa: No
 
     - name: Netflix
       url: https://www.netflix.com/us/
       twitter: Netflixhelps
+      facebook: netflixus
```
Comment (Jawshy) [inline _data/entertainment.yml]: I think [netflix](https://www.facebook.com/netflix) should be used instead as not everyone whom visits twofactorauth.org will be from the US. Thoughts?

File: _data/entertainment.yml [hunk 3]
```diff
@@ -68,36 +75,42 @@ websites:
...
       url: https://soundcloud.com/
       twitter: SCsupport
+      facebook: SoundCloud
       img: soundcloud.png
       tfa: No
 
     - name: Spotify
       url: https://www.spotify.com/
       twitter: spotifycares
+      facebook: spotifyusa
```
Comment (Jawshy) [inline _data/entertainment.yml]: Same comment as with Netflix. I think [Spotify](https://www.facebook.com/Spotify) should be used instead.

**Review by Jawshy — COMMENTED** [review]
Some `facebook` comments and queries.

**Review by Jawshy — APPROVED** [review]
`looks-good`

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**